src/components/AttachmentModal.js

```diff
@@ -74,6 +80,9 @@ const propTypes = {
     /** The report that has this attachment */
     report: reportPropTypes,
 
+    /** The transaction associated with the receipt, if any */
```

Comment:
I think this is slightly clearer

```suggestion
    /** The transaction associated with the receipt attachment, if any */
```